gnd/src/main/java/com/google/android/gnd/repository/DataRepository.java

```diff
@@ -121,13 +121,13 @@ private void onRemoteFeatureVectorChange(DatastoreEvent<Feature> event) {
                     .orElse(Single.error(new DocumentNotFoundException())));
   }
 
-  public Observable<Resource<Record>> getRecordDetails(
+  public Flowable<Resource<Record>> getRecordDetails(
       String projectId, String featureId, String recordId) {
     return getFeature(projectId, featureId)
         .flatMap(feature -> remoteDataService.loadRecordDetails(feature, recordId))
         .map(Resource::loaded)
         .onErrorReturn(Resource::error)
-        .toObservable();
+        .toFlowable();
```

Comment:
Does this compile? I thought BackpressureStrategy was mandatory?

Reply:
@gino-m (should I be @ing you in these comments? or are you already notified?)
It does compile! I thought it was mandatory too, weirdly enough, [Single's toFlowable](http://reactivex.io/RxJava/javadoc/io/reactivex/Single.html#toFlowable) takes no argument. It simply "honors the backpressure of the downstream consumer" which I don't *think* is an issue for us, but I could be wrong.

Reply:
Oooh it's a Single. Doh. Ok LGTM! :)